@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -247,7 +281,8 @@ export class AltitudeBasedClipPlanesEvaluator implements ClipPlanesEvaluator {
         // The near plane calculus is quite straight forward and works the same as for planar
         // projections. We simply search for the closest point of the ground just above
         // the camera, then we apply margin to it along the sphere surface normal:
-        nearPlane = this.getCameraAltitude(camera, projection) - this.nearMargin;
+        const cameraAltitude = this.getCameraAltitude(camera, projection);
+        nearPlane = cameraAltitude - this.maxElevation;
```

Comment:
nearMin is required for any case, we can not create a frustum with near plane distance equal to zero, usually you will be happy with default value, but of course it is configurable. 
As for min/max elevation this gonna be used in conjunction with ElevationRange, evaluated from tiles loaded. It is not yet part of this commit (support for elevation constraints). When this is done, it will be clearly visible in mapview-demo with debug camera on.

Reply:
That is clear, but I just wonder what the cameraAltitude and the maxElevation have to do with each other, why should the near plane have nearMin when the camera has an altitude of 1000m and the maxElevation is 2000m, but if the altitude is 1000m, and the maxElevation is 50 (for example at the sea), then the nearPlane would be 950m, i.e. you have two views with the camera at the same height above the ground, and you have different near planes, to me this is unintuitive.

Reply:
To make my point a bit clear, why does the near plane have to rely on the maxElevation? It should be configured to be as close to the ground, without touching the ground. How high the ground is above the sea level doesn't make any visible difference to the user, hence it should not need to be taken into consideration when computing the near plane.

Reply:
@kkostecki, I understand my mistake. I thought cameraAltitude was the distance above the terrain, but it is the distance above the ground, i.e. the sea level. Sorry for the confusion, I understand what you are doing and this now makes sense.

Reply:
Ok great!

> @kkostecki, I understand my mistake. I thought cameraAltitude was the distance above the terrain, but it is the distance above the ground, i.e. the sea level. Sorry for the confusion, I understand what you are doing and this now makes sense.

Ok exactly as you said in the last comment - camera altitude - above sea level, elevation - relative feature height above sea level - like mountain height.